mteb/tasks/Classification/multilingual/AmazonCounterfactualClassification.py

```diff
@@ -21,7 +21,12 @@ class AmazonCounterfactualClassification(MultilingualTask, AbsTaskClassification
         category="s2s",
         type="Classification",
         eval_splits=["validation", "test"],
-        eval_langs=_LANGUAGES,
+        eval_langs={
+            "en-ext": ["eng-Latn"],
+            "en": ["eng-Latn"],
+            "de": ["deu-Latn"],
+            "ja": ["jpn-Jpan"],
+        },
```

Comment:
I like the dict idea! Do we still need to modify something such that the dict keys are used to fetch the splits from the hf dataset?

Should the values be used for the name in the json result file or the dict key?

Reply:
Yea there will be the need for a few code changes (this is just to make sure we agree on the format beforehand).

Reply:
Thing is if the values are used in the json file that it would be ambiguous if the list has more than 1 item no?

Reply:
Ahh I see what you are getting at. You are referring to {"en-ext": ["eng-Latn"]} vs. {"eng-Latn": "en-ext"}. As a "name" (hf denoted the "en-ext" as "name" when calling the load_dataset).

I was thinking the most intuitive was annotating the "name" with their respective languages (could be multiple), however, we could also do it reverse  {"eng-Latn": "multilingual_subset", "dan-Latn": "multilingual_subset"}. Both approaches are valid.

The first one seems more like annotating the "name".

Reply:
I meant `"en-ext": ["eng-Latn", ...]` vs `"en-ext": "eng-Latn"` - if it's the former and there's additional items in the list do we just select the first one for the results file?

Reply:
(doesn't matter if we leave the results file keys as is, i.e. `en-ext` in that case; it may make more sense to actlly leave results keys as is, since e.g. for this dataset there'd be a clash as it has both `en-ext` & `en` I think)

Reply:
Ahh so list vs. str. Couldn't we imagine that "en-ext" could be multiple languages? (it would be an odd dataset. It would also make the type match in the typing: `Languages | Mapping[str, Languages]`. If not a list it would be `Languages | Mapping[str, Lang]`, which might be a bit confusing, when working with it.

Reply:
Hmm right good question.

Atm. it is the "key", 


I could imagine it looking somewhat like this:

```
{"name": ...
 "scores across subsets": {"en-ext" ...}
 
 # These could essentially be calculated from the above if the scores are loaded with MTEB:
 "scores across language": {"eng_Latn": ...}
 "scores (aggregated)": ...
 }